db/migrate/20230911163251_add_item_to_customer.rb

```diff
@@ -0,0 +1,5 @@
+class AddItemToCustomer < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
You don't need this migration since you have a joins table (this is for a one-to-many association if item belonged to customer